src/aap_eda/core/management/commands/rqworker.py

```diff
@@ -35,12 +41,29 @@ def add_arguments(self, parser: CommandParser) -> None:
 
     def handle(self, *args, **options) -> None:
         if features.DISPATCHERD:
-            self.stderr.write(
+            if "worker_class" not in options:
```

Comment:
@Alex-Izquierdo Can this code block be moved into something like dispatcherd_handle

Reply:
Addressed in https://github.com/ansible/eda-server/pull/1320 please review @mkanoor